### PR TITLE
Updated holidays gem

### PR DESCRIPTION
**Pull request by hassenfratzk**

### CODE REVIEW

File: lib/ach/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ACH
-  VERSION = '0.5.10'.freeze
+  VERSION = '0.6.0'.freeze
```
Comment (nixpulvis) [inline lib/ach/version.rb:2]: `0.5.11` would probably be more appropriate. Don't forget to `git tag x.x.xx` when you're done with this PR. Looks like there's no `0.5.10` tag, but w/e.

Comment (zcotter) [reply lib/ach/version.rb:2]: I would leave the tagging up to the gem maintainer once we get it merged over there

**Comment by zcotter** [comment]
Looks good, can you squash your commits please?

File: lib/ach/version.rb (hunk 1 above)
Comment (nixpulvis) [reply lib/ach/version.rb:2]: @zcotter, agreed. By that logic this PR should touch the version at all.

**Pull request merged**